internal/controller/nemo_entitystore_controller_test.go

```diff
@@ -63,36 +63,36 @@ var _ = Describe("NemoEntitystore Controller", func() {
 			Message: "Required value",
 			Field:   "spec.databaseConfig",
 		}
-		requiredDatabaseHostCause = metav1.StatusCause{
+		requiredCredentialsCause = metav1.StatusCause{
 			Type:    "FieldValueRequired",
 			Message: "Required value",
+			Field:   "spec.databaseConfig.credentials",
+		}
+		invalidDatabaseHostCause = metav1.StatusCause{
+			Type:    "FieldValueInvalid",
+			Message: "Invalid value: \"\": spec.databaseConfig.host in body should be at least 1 chars long",
```

Comment:
Nit: how about "cannot be empty" instead of "at least 1 chars long"

Reply:
This is a standard error from K8s on CRD validation